src/main/java/de/dennisguse/opentracks/chart/ChartFragment.java

```diff
@@ -99,6 +100,50 @@ public void onSharedPreferenceChanged(SharedPreferences sharedPreferences, Strin
                     });
                 }
             }
+
+            if (PreferencesUtils.isKey(R.string.chart_display_elevation_key, key)) {
+                boolean showElevation = PreferencesUtils.shouldShowElevation();
+                if (showElevation != viewBinding.chartView.getShowElevation()) {
+                    viewBinding.chartView.setShowElevation(showElevation);
+                    viewBinding.chartView.applyShowElevation();
```

Comment:
Why do we need `applyShowElevation()`?
This can be done directly in `setShowElevation()`.

This affects also the other two `applyXXX`-methods you introduced.

Reply:
@dennisguse I was thinking it would make sense that the `set`tter method only modifies the value itself, while `apply` could handle any other kind of processing that is related to the current value. But I'm not opposed to rolling it in - I was mostly following the existing convention for `applyReportSpeed`.